README.md

```diff
@@ -780,6 +788,17 @@ If you need the actual *build time* then simply use the a filtered properties fi
 git.build.time=${git.build.time}
 ```
 
+Note on the git.build.number module
+ ------------------------------------
+ Be aware that *git.build.number* is a project specific build number and *git.build.number.unique* contains the server wide
```

Comment:
Rather reword this to explain what it is and where it is available please.
E.g.: 

```
The `git.build.number` variables are available on some hosted CIs and can be used to identify the "number" of the build. Refer to the table below to see which values are supported by which CIs
```

Sounds better?

Note also that the name should be wrapped in ` and not *.